info/demo/index.html

```diff
@@ -0,0 +1,3 @@
+---
+redirect_to: http://help.openmicroscopy.org/demo-server.html
```

Comment:
that page redirects to https://www.openmicroscopy.org/explore/
so it does not really make sense

Reply:
Would you redirect to `/explore` directly? Can you make a suggestion so that we can move forward with this PR?